packages/tempo/doc/tempo.ticker.md

```
@@ -4,18 +4,26 @@
-import { Ticker } from '@magmacomputing/tempo/plugins/ticker';
-import { Tempo } from '@magmacomputing/tempo';
+// Pattern B: Explicit Module (Recommended)
+import { Tempo } from '@magmacomputing/tempo/core';
+import { TickerModule } from '@magmacomputing/tempo/ticker';
 
-// Ticker is now available via Tempo.ticker() in both patterns!
+Tempo.extend(TickerModule);
+```
+
+### Direct Access
+If you need to access the [Reporting & Registry](#reporting--registry) API (like `Ticker.active`), you should import the `Ticker` namespace:
```

markdownlint-cli2 (0.22.0)

[warning] Link fragments should be valid

(MD051, link-fragments)